### PR TITLE
Added in CFU/mL, CFU/g, ng/mg & pg/mg

### PR DESCRIPTION
**Pull request by Toby-Broom**
I break it up in to smaller pieces, sorry about the git mistakes.

### CODE REVIEW

**Comment by fkleedorfer** [comment]
>sorry about the git mistakes.

Not at all. Sorry about the confusion I created. I think I should not have rebased your branch on main and then force-pushed but rather merged main into your branch and then pushed. It makes no difference in terms of the version history as we are going to, eventually, squash all commits in the PR into one.

In any case, as you allow us write access to your branch, it is necessary for you to pull any changes that may have happened on your branch before committing any local changes you have made since your last push. 

It might happen that you make a commit locally before pulling in the latest changes, which will normally be resolved by git but might lead to a situation in which merge conflicts need to be resolved before you can push again. Force-push should be avoided in our scenario (sorry for having done it).

**Comment by steveraysteveray** [comment]
@Toby-Broom, since I just merged a different PR, your branch now needs to be updated. This is straightforward on the GitHub website - just click on the "Update branch" button shown in this screenshot, making sure it is using the "merge" option rather than the "rebase" option:
image

...then you need to git pull to your local branch.

File: src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
Comment (fkleedorfer) [inline src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl:4499]: @steveraysteveray @jhodgesatmb @ralphtq I believe a dimension vector can only have D1 if all other dimensions are 0. right?

File: src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
Comment (fkleedorfer) [inline src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl:4522]: same dimension vector issue as above

**Review by fkleedorfer — CHANGES_REQUESTED** [review]

File: src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
Comment (steveraysteveray) [reply src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl:4499]: Correct.

Comment (fkleedorfer) [reply src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl:4499]: I guess it has to be qudt:hasDimensionVector `qkdv:A0E0L0I0M-1H0T0D0` then.

Comment (steveraysteveray) [reply src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl:4499]: Yes, for the -PER-GM one, and similar correction for the -PER-MilliL one.

**Review by steveraysteveray — APPROVED** [review]

**Comment by steveraysteveray** [comment]
@Toby-Broom, we are so close! The last thing I need your help with is to do step 10 from our updated instructions [here](https://github.com/qudt/qudt-public-repo/wiki/GoodGitPractices). I cannot do this for you because I don't have permission to make changes to your fork. Basically our updated automated build process insists on re-serializing the file before merging.

Let me know if you need help. If you give me access permission to your fork I can do it.

**Comment by Toby-Broom** [comment]
I'm not sure Maven is setup correctly for me, I don't see the workflows. I added you and Florian in to my fork.

**Review by steveraysteveray — APPROVED** [review]

**Comment by steveraysteveray** [comment]
Aargh. We need to wait for @fkleedorfer to approve his requested changes. Then at last we can merge!

**Review by fkleedorfer — APPROVED** [review]

**Pull request merged**
